chromium/pages/cancel/ux.js

```diff
@@ -38,3 +39,23 @@ function displayURL() {
     }
   });
 }
+
+function displayWaybackMachineLink(originURL) {
+  const waybackMachineURLLink = document.getElementById("waybackMachineURL");
+  waybackMachineURLLink.innerText = chrome.i18n.getMessage("cancel_he_blocking_internet_archive_link");
+  waybackMachineURLLink.addEventListener("click", async function () {
+    const response = await fetch("http://archive.org/wayback/available?url=" + encodeURI(originURL));
```

Comment:
`await fetch("http://archive.org/wayback/`

I'm probably misunderstanding this part, but why is `http://` instead of `https://`?

Reply:
Good catch! (I didn't notice this mistake because the link was rewritten to https anyway.) I have pushed a fix. Thanks.